java/jdbc/ConnectionSamples/GSSAuthSSPIConnectSample.java

```diff
@@ -0,0 +1,135 @@
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+/*
+  DESCRIPTION
+  This sample shows how to use SSO uging Kerberos in Windows.
+  Since WIN2019 allowtgtsessionkey registry key is not available 
+  and the only option how to acces Kerberos TGT is via Java's SSPI bridge.
+	
+  PREREQUISITIES
+  - Confirure Kerberos authentication for Oracle database as describe here:
+  https://blog.pythian.com/part-4-implementing-oracle-database-single-sign-on-using-kerberos-active-directory-and-oracle-cmu/
+      
+  - Create DB user identified extenally as:
+  CREATE USER <AD LOGIN> IDENTIFIED EXTERNALLY AS '<AD LOGIN>@<AD REALM>';
+  GRANT CONNECT TO <AD LOGIN>;
+     
+  - Check your Windows has generater Kerb tickets during logon
+  klist tgt
+  klist
+     
+  - Connect to database using this program. 
+  Java's SSPI bridge(sspi_bridge.dll) should be used to renerate required Kerberos ticket for SSO.
+
+   NOTES
+   Use JDK 13 and above on Windows. Check presence of sspi_bridge.dll in JDK intalation
+
+ MODIFIED    (MM/DD/YY)
+ ibre5041    18/04/2025 - Creation
+*/
+
+package kerb;
+
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+import java.util.Properties;
+
+import org.ietf.jgss.GSSCredential;
+import org.ietf.jgss.GSSManager;
+import org.ietf.jgss.GSSName;
+import org.ietf.jgss.Oid;
+
+import oracle.jdbc.OracleConnection;
+import oracle.jdbc.OracleConnectionBuilder;
+import oracle.jdbc.pool.OracleDataSource;
+import oracle.net.ano.AnoServices;
+
+public class GSSAuthSSPIConnectSample {
+    // This should return your AD LOGIN
+    String username = System.getProperty("user.name");
+    // This should return your AD KERBEROS REALM
+    String domain = System.getenv("USERDNSDOMAIN");
+	// Your Database JDBC URL here
+    String url = "jdbc:oracle:thin:@//dbhost1:1521/DBSERVICE";
+
+    public GSSAuthSSPIConnectSample() {
+    }
+
+    public void doit() throws Exception
+    {
+	// Use env variable SSPI_BRIDGE_TRACE=1 in order to trace Java's sspi_bridge.dll plugin
+	// export SSPI_BRIDGE_TRACE=1
+
+	// Various useful tracing options
+	// System.setProperty("oracle.jdbc.Trace", "true");
+	// System.setProperty("sun.security.krb5.debug", "true");
+	// System.setProperty("sun.security.spnego.debug", "true");
+	// System.setProperty("sun.security.jgss.debug", "true");
+	// System.setProperty("java.security.debug", "true");
+	// System.setProperty("sun.security.nativegss.debug", "true");
+
+	// Activate SSPI bridge, your Kerberos token will be created using Windows SSPI API
+	System.setProperty("sun.security.jgss.native", "true");
+	System.setProperty("sun.security.jgss.lib", "sspi_bridge.dll");
```

Comment:
According to [JDK-8214079](https://bugs.openjdk.org/browse/JDK-8214079) there is no need to sun.security.jgss.lib as long as sun.security.jgss.native is set to true. Can you please doublecheck?